### PR TITLE
Import callbacks so that they will be called (Structuring a Multi-Page App flat project layout example)

### LINKED ISSUE #395: Callbacks are not called in Structuring a Multi-Page App flat project layout example
After selecting a dropdown value from either App1 or App2, the selected value is not displayed as intended.

### PR DESCRIPTION
**Pull request by constantlycoding**
Fixes #395 

Currently, callbacks are not called because they are not imported yet.

### CODE REVIEW

**Comment by ned2** [comment]
Thanks so much for the PR!

**Comment by constantlycoding** [comment]
Moved import to `index.py`. Thank you for sorting out the project structure for us!

**Comment by constantlycoding** [comment]
oops

File: tutorial/urls.py [hunk 1]
```diff
@@ -386,7 +386,7 @@ def display_value(value):
 
 from app import app
 from apps import app1, app2
-
+import callbacks
```
Comment (ned2) [inline tutorial/urls.py]: This shouldn't be added to this project structure, just the second flat project structure. Can you remove this one?

Comment (constantlycoding) [reply tutorial/urls.py]: Removed. Was careless.

**Comment by ned2** [comment]
Thanks @constantlycoding!

**Pull request merged**